app/javascript/components/rooms/room/join/RoomJoin.jsx

```diff
@@ -163,10 +163,10 @@ export default function RoomJoin() {
       {
         env.OPENID_CONNECT ? (
```

Comment:
Don't we need to redirect users back to the join page for these cases?

Reply:
Yeah that's a good point - I'll fix this up